less/amount-currency-select.less

```diff
@@ -0,0 +1,8 @@
+.amount-currency-select-btn {
```

Comment:
Literally last comment, sorry...

Could we have the media query nested inside the `.amount-currency-select-btn` selector? One potential mistake point less.

Reply:
no worries, love comments as long as I know what to do :D 
Waiting for that push force to come up here

Reply:
Suprised that grunt did not update the bootstrap of that change, does it matter?

Reply:
@OFFLlNE It's fine, a media query in a selector is not allowed in CSS, but is (and often is preferred) in LESS. Therefore, it compiles to the same thing (media query outside the selector).